generated/nidaqmx/constants.py

```diff
@@ -691,11 +703,11 @@ class ShuntCalSelect(Enum):
 
 
 class ShuntElementLocation(Enum):
-    R_1 = 12465
-    R_2 = 12466
-    R_3 = 12467
-    R_4 = 14813
-    NONE = 10230
+    R1 = 12465  #: xx
+    R2 = 12466  #: xx
+    R3 = 12467  #: xx
+    R4 = 14813  #: xx
+    NONE = 10230  #: xx
```

Comment:
I'm fine with either old or new, but lets delete placeholder comments like this.

Reply:
`ShuntElementLocation `enum is not used in nidaqm-python. Hence, it is removed.